packages/robo/src/cli/utils/cloudflared.ts

```diff
@@ -200,3 +329,87 @@ function download(url: string, to: string, redirect = 0): Promise<string> {
 		request.end()
 	})
 }
+
+async function cloudflareRequest(endpoint: string, method: 'GET' | 'POST' | 'PUT' = 'GET', body: CloudflareBody | TunnelConfig | null = null): Promise<CloudflareResponse> {
+	const response = await fetch(`${CLOUDFLARE_API}${endpoint}`, {
+		method,
+		headers: {
+			'Authorization': `Bearer ${process.env.CLOUDFLARE_API_KEY}`,
+			'Content-Type': 'application/json',
+		},
+		body: body ? JSON.stringify(body) : null,
+	});
+
+	const data: CloudflareResponse = await response.json();
+
+	if (!response.ok || !data.success) {
+		cloudflareLogger.error(`Cloudflare API request failed: ${data.errors[0]?.message || 'Unknown error'}`);
+	} else {
+		cloudflareLogger.debug(`Cloudflare API request succeeded: ${endpoint}`)
+	}
+
+	return data;
+}
+
+async function updateTunnelConfig(id: string, accountId: string) {
+	const tunnelConfig: TunnelConfig = {
+		"config": {
+			"ingress": [
+				{
+					"hostname": `robo.${process.env.CLOUDFLARE_DOMAIN}`,
+					"service": "http://localhost:3000"
+				},
+				{
+					"service": "http_status:404"
+				}
+			]
+		}
+	}
+
+	const tunnelConfigResponse = await cloudflareRequest(`/accounts/${accountId}/cfd_tunnel/${id}/configurations`, 'PUT', tunnelConfig)
+
+	if (tunnelConfigResponse.success) {
+		cloudflareLogger.debug(`Tunnel config updated: ${JSON.stringify(tunnelConfigResponse.result)}`);
+	} else {
+		cloudflareLogger.error(`Failed to update tunnel config: ${JSON.stringify(tunnelConfigResponse.errors)}`);
+	}
+}
+
+function updateEnvFile(key: string, value: string) {
+	const regex = new RegExp(`^${key}=.*$`, 'm');
+	let envContent = fs.readFileSync(path.join(process.cwd(), '.env'), 'utf8');
+
+	if (regex.test(envContent)) {
+		envContent = envContent.replace(regex, `${key}="${value}"`);
+	} else {
+		envContent += `\n${key}="${value}"`;
+	}
+
+	fs.writeFileSync(path.join(process.cwd(), '.env'), envContent, 'utf8');
+	cloudflareLogger.debug(`Updated .env file with ${key}=${value}`)
+}
+
+function installService(token: string) {
```

Comment:
Do we really want to install `cloudflared` as a service on the user machine even though `startCloudflared` already runs the tunnel associated with that token?